client/posts/embed.ts

```diff
@@ -35,7 +43,7 @@ const formatters: { [key: number]: (s: string) => string } = {}
 // Map of providers to information fetcher functions
 const fetchers: { [key: number]: (el: Element) => Promise<void> } = {}
 
-for (let p of ["Youtube", "SoundCloud", "Vimeo"]) {
+for (let p of ["Youtube", "Hooktube", "SoundCloud", "Vimeo"]) {
```

Comment:
These are called through `noembed.com`. You need a separate handler (or 2?) for hooktube. Some sort of error handling would be nice too.